lib/auth.js

```diff
@@ -12,7 +12,10 @@ var restify = require('restify');
 var util = require('util');
 
 var AUTH_TOKEN_HEADER = 'x-adminui-token';
-
+var LOGIN_ATTEMPTS = 5;
+var LOCKED_TIME = 4 * 60 * 60;
+var USERNAME_LOCKED_TIMES = {};
```

Comment:
It's fine to keep globals in all-caps so that they are visibly different from local variables.  If you're not going to keep the globals in capital letters, please at least prefix them with `g_`, e.g.

```
var g_userLockTimes = {};
var g_userLoginAttempts = {};
```